src/player.cpp

```diff
@@ -9111,6 +9106,15 @@ void player::use_fire(const int quantity)
     } else if (has_charges("zweifire_off", quantity)) {
         use_charges("zweifire_off", quantity);
         return;
+    } else if (has_active_bionic("bio_tools") && ( power_level > (quantity * 5) )) {
+        charge_power( -(quantity * 5) );
```

Comment:
Because I have terminal parenthesitis. Yeah, for some reason I overdo them, even when normally typing. Will get rid of them.